charts/fleet/templates/service.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
 {{- if .Values.serviceAnnotations }}
   annotations:
   {{- with .Values.serviceAnnotations}}
-  {{ toYaml . | trim | indent 2}}
+  {{- toYaml . | trim | nindent 4}}
```

Comment:
Typo

Reply:
`nindent` isn't a typo but rather the fix for the bug.  See https://helm.sh/docs/chart_template_guide/function_list/#nindent